src/lib/sql/sql_translator.hpp

```diff
@@ -112,7 +112,7 @@ class SQLTranslator final {
 
   std::shared_ptr<AbstractLQPNode> _translate_create(const hsql::CreateStatement& create_statement);
 
-  std::shared_ptr<AbstractLQPNode> _translate_drop(const hsql::DropStatement& update_statement);
```

Comment:
(comment) does clang-tidy actually find this,or was that you?

Reply:
Mismatching parameter name.